pkg/cloudprovider/providers/azure/azure_fakes.go

```diff
@@ -474,7 +473,6 @@ func (fNSG *fakeAzureNSGClient) List(ctx context.Context, resourceGroupName stri
 }
 
 func getRandomIPPtr() *string {
-	rand.Seed(time.Now().UnixNano())
 	return to.StringPtr(fmt.Sprintf("%d.%d.%d.%d", rand.Intn(256), rand.Intn(256), rand.Intn(256), rand.Intn(256)))
```

Comment:
The later rand still calls 'math/rand', why changing this line?
even delete pkg k8s.io/apimachinery/pkg/util/rand, test in this file can still run well.

Reply:
Each place that uses the global rand should not seed it, it should rely on it being initialized in the `main()`/`MainTest()`. A somewhat better approach would be to explicitly use an instance of a rand that is provided by the caller to make the dependency explicit, but that is a big change that is not worth it.
This PR just cleans up `rand` being seeded all over the place.

Reply:
This is unittest, which 'TestMain' is called for it?